modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/sorting.spec.ts

```diff
@@ -48,15 +48,114 @@ test.describe('Sorting in Data Set Manager', () => {
 				dataSetLabel,
 			});
 		});
+	});
 
-		await test.step('Open new sort modal', async () => {
-			await sortingPage.openAddSortingModal();
+	test('Save and cancel buttons are not present @LPD-9468', async ({
+		page,
+	}) => {
+		await test.step('Check that save and cancel buttons are not present', async () => {
+			await expect(
+				page.getByRole('button', {name: 'Save'})
+			).not.toBeVisible();
+			await expect(
+				page.getByRole('button', {name: 'Cancel'})
+			).not.toBeVisible();
+		});
+	});
+
+	test('Sorting options can be reordered and changes are persisted @LPD-9468', async ({
+		dataSetManagerApiHelpers,
+		sortingPage,
+	}) => {
+		await test.step('Create sorting options', async () => {
+			await dataSetManagerApiHelpers.createDataSetSort({
+				dataSetERC,
+				defaultValue: true,
+				fieldName: 'id',
+				label_i18n: {en_US: 'ID'},
+				orderType: 'asc',
+			});
+
+			await dataSetManagerApiHelpers.createDataSetSort({
+				dataSetERC,
+				defaultValue: false,
+				fieldName: 'name',
+				label_i18n: {en_US: 'Name'},
+			});
+
+			await dataSetManagerApiHelpers.createDataSetSort({
+				dataSetERC,
+				defaultValue: false,
+				fieldName: 'dateCreated',
+				label_i18n: {en_US: 'Date Created'},
+			});
+		});
+
+		await test.step('Navigate to Sorting section', async () => {
+			await sortingPage.goto({
+				dataSetLabel,
+			});
+		});
+
+		await test.step('Check that "Date Created" is below "Name"', async () => {
+			const tableLabelCellTexts =
+				await sortingPage.getTableLabelCellTexts();
+
+			expect(tableLabelCellTexts).toStrictEqual([
```

Comment:
TIL there is a `toStrictEqual`. We commonly use just `toEqual` for string array comparisons. Looking at [docs](https://playwright.dev/docs/api/class-genericassertions#generic-assertions-to-strict-equal), this seems like a slight overkill, but not wrong by any means. Is there a specific case here where strictness is relevant?

Reply:
Ah I actually just assumed `toStrictEqual` was what I should be using for non-string comparisons. So I'll switch these to `toEqual` since it doesn't need to be strict. 

I originally used `toBe` and I thought `toEqual` was the same which is why the error told me I needed `toStrictEqual`.
![Screenshot 2024-07-05 at 7 45 55 AM](https://github.com/liferay-frontend/liferay-portal/assets/4496722/90b8c667-9c3f-4660-b9d5-7d7979ee945d)